gazebo_plugins/include/gazebo_plugins/gazebo_ros_ft_sensor.hpp

```diff
@@ -0,0 +1,84 @@
+// Copyright 2012-2014 Open Source Robotics Foundation
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#ifndef GAZEBO_PLUGINS__GAZEBO_ROS_FT_SENSOR_HPP_
+#define GAZEBO_PLUGINS__GAZEBO_ROS_FT_SENSOR_HPP_
+
+#include <gazebo/common/Plugin.hh>
+
+#include <memory>
+
+namespace gazebo_plugins
+{
+class GazeboRosFTSensorPrivate;
+
+/// This is a controller that simulates a 6 dof force and torque sensor on link or joint.
+/// For joints, the wrench is reported in the joint child link frame and the
+/// measure direction is child-to-parent link. (Force and Torque Feed Back Ground Truth)
+/// If <body_name> is specified, the plugin acts as sensor on a link, otherwise if
+/// <joint_name> is specified, it acts as a sensor on a joint
+/*
+ * \author  John Hsu
+ * \author Francisco Suarez-Ruiz
+ */
+/**
+  Example Usage:
+  \code{.xml}
+    <plugin name="gazebo_ros_ft_sensor" filename="libgazebo_ros_ft_sensor.so">
+
+      <ros>
+
+        <!-- Add a namespace -->
+        <namespace>/demo</namespace>
+
+        <!-- Remap the default topic -->
+        <argument>wrench:=wrench_demo</argument>
+
+      </ros>
+
+      <!-- Link name -->
+      <body_name>link</body_name>
+
+      <!-- Frame id of published message -->
```

Comment:
This can only be used in conjunction with `body_name`, right? If that's correct, mind documenting this here?